webserver/src/layouts/Footer.jsx

```diff
@@ -3,19 +3,26 @@ export default function Footer() {
 
     return (
         <footer className="bg-gray w-full mt-auto">
-            <div className="w-full mx-auto max-w-screen-xl p-4 flex items-center justify-center">
+        <div className="w-full mx-auto max-w-screen-xl p-4 flex items-center justify-center">
 
-        <span className="text-sm text-primary text-center justify-center dark:text-gray-400">
-            Jordan Dialpuri, Haroldas Bagdonas and  Jon Agirre
-            <br/>
-            © 2023 University of York. All Rights Reserved.
-            <br/>
-             <a href="https://www.york.ac.uk/about/legal-statements/">Disclaimer</a>
-             <img className="w-48 mx-auto" src="uoy_logo.png"/>
+            <span className="text-sm text-primary text-center justify-center dark:text-gray-400">
+                Jordan Dialpuri, Haroldas Bagdonas and  Jon Agirre
```

Comment:
Pedantic, but, is the double space after 'and' intended? On my screen it reads 'and  Jon Agirre'.

Reply:
Didn't notice that before the merge, will sort it in the next PR